src/System.Security.Cryptography.OpenSsl/src/System.Security.Cryptography.OpenSsl.builds

```diff
@@ -3,10 +3,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Security.Cryptography.OpenSsl.csproj">
-      <OSGroup>Windows_NT</OSGroup>
+      <OSGroup>Unix</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.OpenSsl.csproj">
-      <OSGroup>Unix</OSGroup>
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
These edits seem a bit unnecessary.  As of right now I don't think we need to split the build, since there don't seem to be any bugfixes that we made in this library for netstandard1.6-consumers.